### PR TITLE
crypto-square: add exercise

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: exercises/practice/crypto-square/.meta/example.gleam [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  |> string.to_graphemes()
+  |> list.filter(set.contains(valid_characters, _))
+}
+
+fn square_width(characters: List(String)) -> Int {
+  assert Ok(sqrt) =
+    characters
+    |> list.length()
+    |> int.to_float()
+    |> float.square_root()
```
Comment (jiegillet) [inline exercises/practice/crypto-square/.meta/example.gleam:43]: There is a `int.square_root` you could use

File: exercises/practice/crypto-square/.meta/example.gleam [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  |> string.join(" ")
+}
+
+fn rows(characters: List(String)) -> List(List(String)) {
+  let width = square_width(characters)
+
+  characters
+  |> list.sized_chunk(into: width)
+  |> list.map(fn(row) {
+    list.append(row, list.repeat(" ", width - list.length(row)))
```
Comment (jiegillet) [inline exercises/practice/crypto-square/.meta/example.gleam:22]: You could use `string.pad_right` somewhere maybe?

File: config.json [hunk 3]
```diff
@@ -1074,6 +1074,24 @@
...
       },
+      {
+        "slug": "crypto-square",
+        "name": "Crypto Square",
+        "uuid": "5bf01e81-b30a-4210-b1ed-1edbf3d28ab1",
+        "practices": [
+          "int",
+          "float",
+          "list",
+          "string"
```
Comment (jiegillet) [inline config.json:1085]: Let's reduce the list to one or two concepts. string and float?

**Review by jiegillet — COMMENTED** [review]
Really neat as usual, thank you!
I left some nitpicky comments as usual :D

File: exercises/practice/crypto-square/.meta/example.gleam (hunk 2 above)
Comment (ErikSchierboom) [reply exercises/practice/crypto-square/.meta/example.gleam]: Done

**Pull request merged**